pages/administratorWorkspaces/administratorWorkspaces.html.js

```diff
@@ -0,0 +1,190 @@
+const { html } = require('@prairielearn/html');
+const { renderEjs } = require('@prairielearn/html-ejs');
+const z = require('zod');
+const _ = require('lodash');
+
+const WorkspaceSchema = z.object({
```

Comment:
I'm a bit torn here. I think `Codec` somewhat more accurately reflects the fact that these can actually transform data instead of just validating it (["parse, don't validate"](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/)). On the other hand, Zod refers to these things exclusively as schema; the word "codec" is leftover from our brief time with [`io-ts`](https://github.com/gcanti/io-ts). I guess I'm leaning towards schema in the green field of PrairieLearn?

Reply:
I'm fine with that. I'll probably do a mass-rename in PT to preserve my sanity.

Reply:
😭 sorry for switching! It shouldn't be too much trouble to find/replace this in PT; I'd be happy to do it since I'm the reason for the change.